### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by JantuDeb**
### Navbar
image

### Side Navigation menu
image

### CODE REVIEW

File: src/components/navbar/navbar.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.navbar{
```
Comment (t007rushi) [inline src/components/navbar/navbar.css:1]: can put this file in common CSS file as many components will come

**Review by t007rushi — APPROVED** [review]
Host it

File: src/components/navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    height: 4rem;
+}
+
+.nav-menu{
+    position: absolute;
+    z-index: 10;
```
Comment (tushartiwari7) [inline src/components/navbar/navbar.css:7]: Try using immediate next values for z-indexes and not to skip z-index values.
So, IMHO z-index: 2 or 3 would be a better approach.

**Review by tushartiwari7 — APPROVED** [review]

File: src/components/navbar/navbar.css (hunk 2 above)
Comment (JantuDeb) [reply src/components/navbar/navbar.css:7]: will do that

**Pull request merged**